dkron/api.go

```diff
@@ -119,6 +119,7 @@ func (h *HTTPTransport) APIRoutes(r *gin.RouterGroup, middleware ...gin.HandlerF
 	// Place fallback routes last
 	jobs.GET("/:job", h.jobGetHandler)
 	jobs.GET("/:job/executions", h.executionsHandler)
+	jobs.GET("/:job/execution/:execution", h.executionHandler)
```

Comment:
As @yvanoers said, this will maintain the convention

```suggestion
	jobs.GET("/:job/executions/:execution", h.executionHandler)
```